lib/client/api.go

```diff
@@ -703,7 +703,9 @@ func (tc *TeleportClient) Login() error {
 
 	// generate a new keypair. the public key will be signed via proxy if our password+HOTP
 	// are legit
-	priv, pub, err := native.New().GenerateKeyPair("")
+	keygen := native.New()
+	defer keygen.Close()
+	priv, pub, err := keygen.GenerateKeyPair("")
```

Comment:
I would suggest in the future to make key generation a singleton. we've been calling `native.New()` and leaking the key pool all over. why do we need multiple pools?